Rename product groups: #g-mdm → #g-apple-at-work, #g-software → #g-auto-patching (#48482)
fleetdm/fleet: handbook/customer-success/README.md

    - Have a support engineer collect data on open and closed bugs from the previous quarter and highlight any P0 or P1 incidents along with a summary of the postmortem (search Unthread and GitHub for issues tagged with the customer codename and ':bug').
    - Summarize status updates for open feature requests and highlight delivered feature requests.
    - For managed cloud customers, reach out to #help-infrastructure to collect information on cloud uptime and any outages or alarms.

>     - Provide one slide with information on the latest Fleet release and any upcoming big ticket features which can be found on the product board and current release board for #g-apple-at-work and #g-endpoint-ops

Author: ```suggestion
    - Provide one slide with information on the latest Fleet release and any upcoming big ticket features which can be found on the product board and current release board for any product or working group.
```
